### PR TITLE
Feat(front): add welcome page

### PR DESCRIPTION
**Pull request by Ekeu**
| Q                 | A
| ----------------- | ----------
| Bug fix?          | no
| New feature?      | yes
| BC breaks?        | no     
| Related Issue     | Feat #2 
| Need Doc update   | no

## Describe your change

- Created a Navbar Component
- Created a Footer Component
- Cleaned out the default.vue file in the layout folder and added the
Navbar and Footer Components
- Removed the ssr:false option in the nuxt.config.js file to enable SSR
- Made our app a universal app withe mode:"universal" in the
nuxt.config.js
- Added tailwindcss @nuxtjs/tailwindcss which can be found in the
- Added a tailwind.config.js file with some basic configurations
buildModules
- Cleaned out the default index.vue content
- Added the Hero.vue and ProjectCard.vue Component

## What problem is this fixing?

It isn't solving a problem just an initial Front-End setup

### CODE REVIEW

File: components/Hero.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        <!-- hero text -->
+        <div class="hero-text col-span-6">
+          <h1
+            class=" font-bold text-4xl md:text-5xl max-w-xl text-gray-900 leading-tight"
+          >
+            Making Open Source Look Different
+          </h1>
+          <hr class=" w-12 h-1 bg-nebula rounded-full mt-8" />
+          <p class="text-gray-800 text-base leading-relaxed mt-8 font-semibold">
+            We provide an open platform wer any developer can share, contribute
```
Comment (TomKlotzPro) [inline components/Hero.vue:16]:
```suggestion
            We provide an open platform where any developer, designer or customer can share, contribute
```
Typo + suggestions

File: components/Hero.vue [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            or donate on any open source project!
```
Comment (TomKlotzPro) [inline components/Hero.vue:17]:
```suggestion
            or donate on any open source projects!
```
Typo

**Review by TomKlotzPro — CHANGES_REQUESTED** [review]
Looks awesome!!! Congrats for your PR @Ekeu really good job. 🚀 

I've left some request changes don't hesitate discuss about it on the PR :)

**Comment by Ekeu** [comment]
Thanks for the feedback @TomKlotzPro !

File: components/Hero.vue (hunk 1 above)
Comment (Ekeu) [reply components/Hero.vue:16]: Thanks 👍 .Will be taken into account in my next commit.

File: components/Hero.vue (hunk 2 above)
Comment (Ekeu) [reply components/Hero.vue:17]: Thanks 👍 .Will be taken into account in my next commit.

**Review by TomKlotzPro — APPROVED** [review]
Awesome job you can merge the PR when you want!! 

Congrats @Ekeu ! 🚀

**Comment by Ekeu** [comment]
Thanks we did an awesome job! 💯

**Pull request merged**